test/user.test.js

```diff
@@ -4,17 +4,19 @@ const { TEST_BASE_URL } = require("./constants");
 const testInit = require("./testInit");
 const request = supertest(app);
 
+const registrationDetails = {
+  email: "mailstoeze@gmail.com",
+  name: "Peter",
+  password: "Password",
+}
+
 afterAll(async () => {
   await testInit.tearDown();
 });
 
 describe("User Test", () => {
   it("should create a new user", async () => {
-    const user = await request.post(`${TEST_BASE_URL}/auth/signup`).send({
-      email: "mailstoeze@gmail.com",
-      name: "Peter",
-      password: "Password",
-    });
+    const user = await request.post(`${TEST_BASE_URL}/auth/signup`).send(registrationDetails);
     expect(user.body.error).toBe(false);
     expect(user.body.data.email).toBe("mailstoeze@gmail.com");
```

Comment:
nit: in the case that we have this change, I think `mailstoeze@gmail.com` should be changed to `registrationDetails.email`